### PR TITLE
Export db size, deadlocks and changed row metrics

### PR DESCRIPTION
**Pull request by zaynetro**
## Problem

We want to report metrics for the oldest user database.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [ ] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

File: vm-image-spec.yaml [hunk 1]
```diff
@@ -142,6 +142,32 @@ files:
...
+          select pg_database_size(datname) as db_size, deadlocks,
+                 tup_inserted as inserted, tup_updated as updated, tup_deleted as deleted,
+                 datname
+            from pg_stat_database
+           where datname IN (
+             select datname
+               from pg_database
+              where datname <> 'postgres' and not datistemplate
+              order by oid
+              limit 1
```
Comment (vadim2404) [inline vm-image-spec.yaml:168]: what will happen if we remove this limit?

Average number of databases per branch

```
neon_console=> select avg(cnt) from (select count(*) cnt from databases group by branch_id) a;
        avg
--------------------
 1.1252440084040954
(1 row)

Time: 798.818 ms
```

but there are some extremums 

```
neon_console=> select count(*) cnt from databases group by branch_id order by cnt desc limit 10;
 cnt
-----
 925
 421
 313
 204
 168
 160
 132
 110
 105
  99
(10 rows)
```

Comment (zaynetro) [reply vm-image-spec.yaml:168]: I think for the MVP we need this limit. Without this limit it is too easy to abuse the system by creating a lot of databases.
I am all in to lifting this limit in the future after we understand the needs better.

Comment (vadim2404) [reply vm-image-spec.yaml:168]: may I ask you to leave this comment right in the yaml file?

File: vm-image-spec.yaml [hunk 2]
```diff
@@ -142,6 +142,51 @@ files:
...
+           );
+
+      - metric_name: max_cluster_size
+        type: gauge
+        help: 'neon.max_cluster_size setting'
+        key_labels:
+        values: [max_cluster_size]
+        query: |
+          select setting::int as max_cluster_size from pg_settings where name = 'neon.max_cluster_size';
```
Comment (zaynetro) [inline vm-image-spec.yaml:180]: Do we need `greatest(0, setting::int)` here? In one of my staging databases the setting is -1. I think having 0 in grafana make more sense than -1.

**Review by vadim2404 — APPROVED** [review]

**Comment by vadim2404** [comment]
if you cherry-pick your commit to this PR (https://github.com/neondatabase/neon/pull/7081) it will be released this week

**Pull request merged**